cfme/web_ui/__init__.py

```diff
@@ -1280,6 +1280,7 @@ def _fill_form_list(form, values, action=None, action_always=False):
 
     """
     logger.info('Beginning to fill in form...')
+    sel.wait_for_ajax()
```

Comment:
I believe there is an ajax wait at the end of the function nav.... @mfalesni  ??